rfcs/text/0040-volume-device.md

```diff
@@ -106,6 +281,7 @@ The following are the people that consulted on the contents of this RFC.
 
  * @Trinity2019    | author
  * @ricardoelastic | reviewer
```

Comment:
```suggestion
 * @ricardoungureanu | reviewer
```

nit: reviewer GitHub handle

Reply:
fixed in [825f9c0](https://github.com/elastic/ecs/pull/2229/commits/825f9c038df5e4daf236f5cee7de7a1445216ae6)